specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.2/keys.json

```diff
@@ -1480,10 +1528,34 @@
           "type": "string",
           "description": "The underlying HSM Platform.",
           "readOnly": true
+        },
+        "attestation": {
+          "$ref": "#/definitions/KeyAttestation",
+          "description": "The key or key version attestation information."
         }
       },
       "description": "The attributes of a key managed by the key vault service."
     },
+    "KeyAttestation": {
+      "type": "object",
+      "properties": {
+        "certificates": {
+          "type": "string",
+          "format": "base64url",
+          "description": "The certificates used for validating attestation."
```

Comment:
It's base64url-encoded, sure, but in what format are the certificates before base64url-encoding? PEM? PKCS12? Be specific.

Same probably goes for the `privateKeyAttestation` and `publicKeyAttestation` blobs below, though I imagine those are probably not meant to be parsed by consumers...which begs the question: why return them? Do they just save them like they would a Security Domain blob for MHSM?

Reply:
For the second part of the question, the purpose of returning the attestation blobs.

Yes, customers need to save them and pass the certificate bundle and attestation blobs to a python script. This python script validates the attestation.